charts/shoot-cloud-config/charts/downloader/values.yaml

```diff
@@ -1,2 +1,5 @@
 secretName: cpu-worker-0
-kubeconfig: kubeconfig for cloud-config-script
\ No newline at end of file
+kubeconfig: kubeconfig for cloud-config-script
+images:
+  hyperkube: k8s.gcr.io/hyperkube:1.9.3
```

Comment:
Previously, we used `k8s.gcr.io/hyperkube:v1.12.3`. Why did you change to `1.9.3`?

Reply:
This image is rewritten in func (o *Operation) ComputeDownloaderCloudConfig.
Anyway, I will change it to be repository:tag instead of detailed image repo

Reply:
OK. The problem is that we don't want to have this version dynamic (in the cloud config downloader script). We just need the hyperkube image there to use `kubectl`. We don't want to use the same version as specific in the shoot manifest to prevent unneeded/unwanted node reprovision processes.

Reply:
1. I rebased code and follow the latest version.
2. As for the network issue in Alicloud, I need to change the repo name though the version will keep as v1.12.3. This version will be specified in the code... It will be nicer after the extension solution is done.